aeson.cabal

```diff
@@ -35,9 +35,6 @@ extra-source-files:
     benchmarks/json-data/*.json
     cbits/*.c
     changelog.md
-    examples/*.cabal
```

Comment:
Why remove these?

Reply:
There are two points:
1. I don't think anyone is actually looking for the examples in `.tar.gz`. The tarball is quite well abstracted and hidden by tools like `cabal` and `stack`. I think it's more likely people will find examples through GitHub, than by manually unpacking the tarball and looking there.
2. It may make easier to add examples which are not under BSD-3-Clause license. I thought about e.g. `cabal-plan` implementation of `plan.json`, which is GPLd. One could find something under Apache-2.0 too. Anyway, not worrying about licenses is simpler, and one doesn't need to worry if one doesn't distribute :)

Reply:
Sounds good!